docs/tutorials/01-labeling-finetuning.ipynb

```diff
@@ -746,7 +731,7 @@
    "id": "7703d9a4-bfce-466f-accc-65f65a784f2d",
    "metadata": {},
    "source": [
-    "## 4. Testing the fine-tuned model\n",
+    "## 4. Test the fine-tuned model\n",
```

Comment:
Below you used the present participle ("Adding", "Training"), so i would use it here, too
 ```suggestion
    "## 4. Testing the fine-tuned model\n",
```